extensions/claude-throne/webview/main.js

```
@@ -8,27 +8,286 @@
+      case 'customProviderDeleted':
+      case 'popularModels':
+        // These require payload but we don't validate structure in safe mode
+        if (!payload) {
+          console.error('[Schema Validation] Message type requires payload:', type);
+          return null;
+        }
+        break;
+        
+      default:
+        // Comment 8: Log schema mismatches in DEBUG mode
+        const debugCheckbox = document.getElementById('debugCheckbox');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)